### PR TITLE
Hello Rails and  React  project  Set up

### PR DESCRIPTION
**Pull request by Nasirkhan294**
### In this milestone I:

- [ ] Initialized rails project with webpack.
- [ ] Add GitHub action linters
- [ ] Prepared dummy database for random greetings.
- [ ] Displayed random in the browser.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "app",
+  "private": "true",
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
```
Comment (gemmen29) [inline package.json:5]: - [ ] Kindly fix this error
image
 I get this error when I run `npm run build` to build your project.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (gemmen29) [inline package.json:1]: - [ ] Kindly install the node dependencies as described in the provided tutorial as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=Install%20needed%20node%20dependencies%20as%20described%20in%20the%20previous%20lesson%27s%20tutorial.).

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Nasirkhan294, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Nasirkhan294) [reply package.json:5]: Hi @gemmen29,

Thanks for highlighting the issues. I will fix them and submit them for another evaluation.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Nasirkhan294, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**